content/en/metrics/custom_metrics/guide/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Custom Metrics Guides
+kind: guide
+---
+
+{{< whatsnext desc="General guides:" >}}
+    {{< nextlink href="metrics/custom_metrics/guide/tag-governance-drop-metrics-missing-specific-tags" >}}Custom Metrics Governance - Drop Metrics Missing Specific Tags{{< /nextlink >}}
```

Comment:
Don't forget to also update the filename!

Reply:
😅  Thanks!